raiden/constants.py

```diff
@@ -181,10 +181,10 @@ class RoutingMode(Enum):
 ETH_GET_LOGS_THRESHOLD_SLOW = ETH_GET_LOGS_TIMEOUT // 2
 
 # Keep in sync with .circleci/config.yaml
-HIGHEST_SUPPORTED_GETH_VERSION = "1.9.11"
+HIGHEST_SUPPORTED_GETH_VERSION = "2.0.0"
 LOWEST_SUPPORTED_GETH_VERSION = "1.9.7"
 # this is the last stable version as of this comment
-HIGHEST_SUPPORTED_PARITY_VERSION = "2.7.2"
+HIGHEST_SUPPORTED_PARITY_VERSION = "3.0.0"
```

Comment:
Does Raiden work with openethereum? Or should we use the latest parity version here?

Reply:
sorry, that was not supposed to be part of this PR